library/std/src/sys/itron/error.rs

```diff
@@ -61,7 +61,7 @@ pub fn error_name(er: abi::ER) -> Option<&'static str> {
         abi::E_NOMEM => Some("insufficient memory"),
         abi::E_NOID => Some("no ID number available"),
         abi::E_OBJ => Some("object state error"),
-        abi::E_NOEXS => Some("non-existent object"),
+        abi::E_NOEXS => Some("nonexistent object"),
```

Comment:
I believe these strings are taken from the micro-itron spec, and if so we shouldn't change them.